### ISSUE TITLE
Is this project abandoned?

### ISSUE THREAD

**Issue by kiler129** [jmhobbs/Swiftmailer-Transport--AWS-SES#31]
I see this library wasn't updated for a while. @jmhobbs are you still maintaining this code, since even #25 is open for very long time with no response.

**Comment by jmhobbs** [comment]
Nope!  It's alive, it's just stable so there isn't much to do, and I'm not sure it has a ton of users since SES has an SMTP interface now.

As for #25, you'll note from comments that there is a branch with this feature implemented.  However, I'm not comfortable merging into master until I get some testing feedback.  Since I don't personally send large volumes of mail with this library anymore, I'm dependent on others to loop me back in.

tl;dr It's alive, #25 has a patch but it's not production proven (afaik)

If you have features or bugs, please let me know.  Thanks!
